lib/core/src/Cardano/Pool/DB/Sqlite/TH.hs

```diff
@@ -45,8 +45,7 @@ PoolProduction
     poolProductionEpochNumber    Word64       sql=epoch_number
     poolProductionSlotNumber     Word16       sql=slot_number
     poolProductionPoolId         W.PoolId     sql=pool_id
-    Primary
-        poolProductionEpochNumber
-        poolProductionSlotNumber
+
+    Primary poolProductionEpochNumber poolProductionSlotNumber
```

Comment:
Like... seriously :|

Reply:
thanks @rvl very much for that!!! I was looking completely in different directions :sweat_smile: